python/ray/serve/drivers.py

```diff
@@ -118,104 +111,3 @@ async def get_intermediate_object_refs(self) -> Dict[str, Any]:
     async def get_pickled_dag_node(self) -> bytes:
         """Returns the serialized root dag node."""
         return self.dags[self.MATCH_ALL_ROUTE_PREFIX].pickled_dag_node
-
-
-@PublicAPI(stability="alpha")
-class gRPCIngress:
-    """
-    gRPC Ingress that starts gRPC server based on the port
-    """
-
-    def __init__(self, port: int = DEFAULT_GRPC_PORT):
-        """Create a gRPC Ingress.
-
-        Args:
-            port: Set the port that the gRPC server will listen to.
-        """
-
-        self.server = grpc.aio.server()
-        self.port = port
-
-        self._attach_grpc_server_with_schema()
-
-        self.setup_complete = asyncio.Event()
-        self.running_task = get_or_create_event_loop().create_task(self.run())
-        ServeUsageTag.GRPC_INGRESS_USED.record("1")
-
-    async def run(self):
-        """Start gRPC Server"""
-        logger.info(
-            "Starting gRPC server with on node:{} "
-            "listening on port {}".format(ray.util.get_node_ip_address(), self.port)
-        )
-        address = "[::]:{}".format(self.port)
-        try:
-            # Depending on whether RAY_USE_TLS is on, `add_port_to_grpc_server`
-            # can create a secure or insecure channel
-            self.grpc_port = add_port_to_grpc_server(self.server, address)
```

Comment:
@GeneDer  do we support TLS (secure channel) in the current grpc proxy?

Reply:
Right now it's just using unsecured port https://github.com/ray-project/ray/blob/master/python/ray/serve/_private/proxy.py#L1617 But adding support should be pretty easy.